docs/_docs/troubleshooting.md

```diff
@@ -36,11 +36,12 @@ If the service isn't wanting to restart, look at its status for clues as to why
 
 > If you find yourself constantly needing to restart a service, there's a chance it was never set to autostart on system boot. Use `sudo systemctl enable servicename` to fix this problem.
 
-## 3. Basic troubleshooting
+## 3. Known problems
 
 If you have one of these issues, please try to resolve it with the instructions below before moving on in the 
 troubleshooting checklist.
 
+- **Streaming player on Microsite and Listen player on Dashboard not working?** The problem is caused by a bug in writing to the database during the setup wizard. This can be fixed by going to **Settings** -> **Stream Settings** and toggling the **Default Streaming** and **Custom/ 3rd Party Streaming** option, accepting the popup dialogues, and clicking **Save** at the top of the settings page.
```

Comment:
```suggestion
- **Streaming player on Microsite and Listen player on Dashboard not working?** The problem could be caused by a bug in writing to the database during the setup wizard. This can be fixed by going to **Settings** -> **Stream Settings** and toggling the **Default Streaming** and **Custom/ 3rd Party Streaming** option, accepting the popup dialogues, and clicking **Save** at the top of the settings page.
```

Also if this PR fixes the issue, is this still relevant?

Reply:
There's still an issue reading from/writing to the database during setup, but this makes it possible once toggling the options in Stream Settings.

Reply:
Can you create a issue describing the exact issue?

Reply:
Then this can close #461 and that new issues can track any further problems

Reply:
Sure thing!